__tests__/app.test.js

```diff
@@ -31,18 +31,81 @@ describe('university routes', () => {
   });
 });
 
+// describe('brewery routes', () => {
+//   beforeEach(() => {
+//     return setup(pool);
+//   });
+  
+//   it('/brewery should return a list of breweries', async () => {
+//     const res = await request(app).get('/brewery');
+//     const expected = brewery.map((brew) => {
+//       return { id: brew.id, name: brew.name };
+//     });
+//     expect(res.body).toEqual(expected);
+//   });
+
 describe('brewery routes', () => {
   beforeEach(() => {
     return setup(pool);
   });
   
   it('/brewery should return a list of breweries', async () => {
     const res = await request(app).get('/brewery');
-    const expected = brewery.map((brew) => {
-      return { id: brew.id, name: brew.name };
-    });
-    expect(res.body).toEqual(expected);
+    // const expected = brewery.map((brew) => {
+    //   return { id: brew.id, name: brew.name };
+    // });
+    expect(res.body).toEqual([
```

Comment:
Usually, when we have this much data in a test, we will store it in a whateverData.js file and compare it that way, this keeps the test files shorter and more legible (which is super helpful for when you are revisiting your code after not looking at it for a long time.  What is your reasoning for typing it all out here?